packages/common/src/hooks/useSortBy/useSortBy.ts

```diff
@@ -44,8 +46,5 @@ export const useSortBy = <T extends ObjectWithStringKeys>({
     return { ...newSortBy, direction: getDirection(!!newSortBy?.isDesc) };
   }, []);
 
-  return {
-    sortBy: { ...sortBy, direction: getDirection(!!sortBy.isDesc) },
-    onChangeSortBy,
-  };
+  return { sortBy, onChangeSortBy };
```

Comment:
I need to revist sorting. Decided to leave it here for this PR.

This was returning a new sortBy object everytime, causing annoying re-renders